lte/gateway/c/core/oai/tasks/nas5g/src/M5GAuthenticationResult.cpp

```diff
@@ -26,63 +26,57 @@ AuthenticationResultMsg::AuthenticationResultMsg(){};
 AuthenticationResultMsg::~AuthenticationResultMsg(){};
 
 // Decode Authentication Result Message and its IEs
-int AuthenticationResultMsg::DecodeAuthenticationResultMsg(
-    AuthenticationResultMsg* auth_result, uint8_t* buffer, uint32_t len) {
+int AuthenticationResultMsg::DecodeAuthenticationResultMsg(uint8_t* buffer,
+                                                           uint32_t len) {
```

Comment:
No change in indentation has been made, only the removal of unnecessary field to avoid misuse of method and ease the comprehension of the code.
The indentation is automatically placed in the devcontainer environment.